services/gossip/adapter/direct_transport.go

```diff
@@ -335,8 +335,6 @@ func calcPaddingSize(size uint32) uint32 {
 }
 
 func (t *directTransport) sendKeepAlive(ctx context.Context, conn net.Conn) error {
-	t.logger.Info("sending keepalive", log.String("peer", conn.RemoteAddr().String()))
```

Comment:
why?

Reply:
Because it's polluting the logs and has no real value, so @talkol agreed to kill it. I kept the log that reports if we failed to send the keep alive request.